pkg/apis/apps/v1beta2/defaults_test.go

```diff
@@ -232,6 +232,10 @@ func TestSetDefaultStatefulSet(t *testing.T) {
 							Partition: &defaultPartition,
 						},
 					},
+					PersistentVolumeClaimRetentionPolicy: &appsv1beta2.StatefulSetPersistentVolumeClaimRetentionPolicy{
```

Comment:
it's really surprising the same test in pkg/apis/apps/v1/defaults_test.go didn't need updating